sys-auth/sssd/sssd-2.3.1.ebuild

```diff
@@ -46,20 +51,34 @@ COMMON_DEP="
 	>=dev-libs/cyrus-sasl-2.1.25-r3[kerberos]
 	>=sys-apps/dbus-1.6
 	acl? ( net-fs/cifs-utils[acl] )
+	kcm? (
+		dev-libs/jansson:0=
+		net-misc/curl:0=
+	)
 	nfsv4? ( || ( >=net-fs/nfs-utils-2.3.1-r2 net-libs/libnfsidmap ) )
 	nls? ( >=sys-devel/gettext-0.18 )
 	virtual/libintl
 	netlink? ( dev-libs/libnl:3 )
```

Comment:
Usually I sort these to be all unconditional, then conditional, alphabetically by category and then package name. Up to you/Matt.

Reply:
i like this more too so i did just that (and test already had it that way.